elements/bulbs-video/components/revealed.js

```diff
@@ -247,7 +252,7 @@ export default class Revealed extends React.Component {
 
     player.setup(playerOptions);
 
-    GoogleAnalytics.init(player, videoMeta.gaPrefix);
+    GoogleAnalytics.init(player, videoMeta.sendAnalyticsEvent);
```

Comment:
I'm pro passing in the tracking function into the plugin. But per my other comment I think both of these should be valid:

```js
GoogleAnalytics.init(player, global.ga);
```

or

```js
GoogleAnalytics.init(player, prefixedGa);
```